modules/core/shared/src/main/scala/lucuma/core/math/syntax/units.scala

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) 2016-2025 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package lucuma.core.math.syntax
```

Comment:
I think we are concentranting all syntax extensions in `lucuma.core.syntax`.

Reply:
We have e.g.
`import lucuma.core.math.syntax.int.*`

Reply:
an exercise for a newcomer